yorkie/src/main/kotlin/dev/yorkie/util/ConnectRpcErrorUtil.kt

```diff
@@ -0,0 +1,22 @@
+package dev.yorkie.util
+
+import com.connectrpc.Code
+import com.connectrpc.ConnectErrorDetail
+import com.connectrpc.ConnectException
+
+/**
+ * `isRetryable` will return true if the given error is retryable.
```

Comment:
you can improve the documentation comment using `[isRetryable]`.

Reply:
Updated comment. Thanks.